stubs/livewire/resources/views/api/api-token-manager.blade.php

```diff
@@ -39,7 +39,7 @@
                 Created.
             </x-jet-action-message>
 
-            <x-jet-button wire:loading.class="opacity-25" wire:loading.attr="disabled">
+            <x-jet-button>
```

Comment:
Livewire automatically disables submit buttons on form submission. Also, because we added `disabled:opacity-25`, we don't need to worry about that either.